### ISSUE TITLE
http://idlelivelink.blogspot.com/p/golink.html?url=aHR0cHM6Ly9maWxlcy5zZWN1cmVzZXJ2ZXIubmV0LzRzZkRxUnAzT0k1QTZy

### ISSUE THREAD

**Issue by kartikbilla007** [reek/anti-adblock-killer#3242]
![image](https://cloud.githubusercontent.com/assets/16859254/26387812/4badcae0-4005-11e7-87e8-42b4526bbcf1.png)

**Comment by uBlock-user** [comment]
uBO - `idlelivelink.blogspot.com##script:inject(abort-on-property-read.js, matchMedia)`

@jspenguin2017 5 sec timer and context menu blocking needs to be patched.

**Comment by jspenguin2017** [comment]
I can't reproduce the block screen.

**Comment by jspenguin2017** [comment]
I removed the timer.

**Comment by uBlock-user** [comment]
> I can't reproduce the block screen.

Yes the block screen is already patched by the Script. What about the context menu ?

**Comment by jspenguin2017** [comment]
Didn't notice that, it's now patched.
